WFP-VAM/prism-app
# Improve failed Kobo authentication behavior in the frontend; COUNTRY=cambodia

This fixes issue https://github.com/WFP-VAM/prism-app/issues/560 and https://github.com/WFP-VAM/prism-app/issues/910.

How to test the feature:

- use country `cambodia` 
- from field reports use flood report
- enter incorrect kobo credentials
- see that the auth modal is still showing and "Authentication failed" message appears
- the layer is turned off when cancel is click on the auth modal

Screenshot/video of feature:
![Screenshot from 2023-11-14 10-23-44](https://github.com/WFP-VAM/prism-app/assets/80451946/2d313b0e-a4ae-4533-ae12-ef75bfc0e1d6)
